feature/container/containerz/tests/container_lifecycle/containerz_test.go

```diff
@@ -2,45 +2,73 @@ package container_lifecycle_test
 
 import (
 	"context"
-	"crypto/tls"
 	"flag"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 	"testing"
 	"time"
 
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/openconfig/containerz/client"
-	cpb "github.com/openconfig/featureprofiles/internal/cntrsrv/proto/cntr"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/credentials"
-	"google.golang.org/grpc/credentials/insecure"
+
+	cpb "github.com/openconfig/gnoi/containerz"
 )
 
 var (
 	containerTar        = flag.String("container_tar", "/tmp/cntrsrv.tar", "The container tarball to deploy.")
 	containerUpgradeTar = flag.String("container_upgrade_tar", "/tmp/cntrsrv-upgrade.tar", "The container tarball to upgrade to.")
-	containerzAddr      = flag.String("containerz_addr", "localhost:19999", "containerz server address")
 )
 
 const (
 	instanceName = "test-instance"
 )
 
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
 func containerzClient(ctx context.Context, t *testing.T) *client.Client {
-	client.Dial = func(ctx context.Context, target string, opts ...grpc.DialOption) (conn *grpc.ClientConn, err error) {
-		return grpc.DialContext(ctx, target, grpc.WithTransportCredentials(insecure.NewCredentials()))
-	}
-	cli, err := client.NewClient(ctx, *containerzAddr)
-	if err != nil {
-		t.Fatalf("unable to dial containerz: %v", err)
+
+	dut := ondatra.DUT(t, "dut")
+	switch dut.Vendor() {
+	case ondatra.ARISTA:
+		if deviations.ContainerzOCUnsupported(dut) {
```

Comment:
Can we file an internal bug to make sure that we have OC modelling for the parameters that we need here (and across vendors)?